faris-wireframe-userstory/public/main.css

```diff
@@ -0,0 +1,143 @@
+body,html {
+    margin: 0;
+    padding: 0;
+    font-family: Verdana;
+  }
+p {
+    padding-right: 70px;
+    word-break: break-word;
+}
+/* Flex box css */
+.container {
+    display: flex;
+    justify-content: flex-start;
+    flex-wrap: nowrap;
+    margin-left: 100px;
+    margin-right: 100px;
+    height: 400px;
+}
+.form-container {
+    display: flex;
+    justify-content: center;
+    flex-wrap: nowrap;
+    margin-left: 100px;
+    margin-right: 100px;
+    margin-bottom: 50px;
+}
+.footer-container {
+    display: flex;
+    justify-content: flex-start;
+    flex-wrap: nowrap;
+    margin-left: 100px;
+    margin-right: 100px;
+}
+.item {
+    width: 100%;
+}
+#nav {
+    z-index: 2;
+    margin-top: 80px;
+}
+footer button {
```

Comment:
The `<footer></footer>` element can be used on multiple places across the website. So it won't be smart to do it like this. Use classes instead.